frontend/src/Onboarding/RedirectScreen.tsx

```diff
@@ -27,6 +29,33 @@ export const RedirectScreen: React.FC<Props> = ({ redirectUrl }) => {
     boolean | undefined
   >();
 
+  const calculateAcademicYear = (completedCourses: ScheduleCourse[]) => {
+    // sort the courses from the earliest to lastest semester
+    if (completedCourses && completedCourses.length != 0) {
+      const sortedCourses = JSON.parse(JSON.stringify(completedCourses)).sort(
+        (first: ScheduleCourse, second: ScheduleCourse) => {
+          if (Number(first.semester!) < Number(second.semester!)) {
+            return -1;
+          } else if (Number(second.semester!) < Number(first.semester!)) {
+            return 1;
+          } else {
+            return 0;
+          }
+        }
+      );
+
+      const earliestSemesterYear = sortedCourses[0].semester?.substring(0, 4);
+      const latestSemesterYear = sortedCourses[
+        sortedCourses.length - 1
+      ].semester?.substring(0, 4);
+      const academicYear =
+        Number(latestSemesterYear) - Number(earliestSemesterYear) + 1;
```

Comment:
Add 1 because the semester year from Khoury is based on the Spring semester (so Fall 2018-Spring 2019 would be so Fall 2019, and Spring 2019).So, for a student taking classes 2018-2022: `2022-2019 +1 = 4 `